python/pyspark/sql/tests.py

```diff
@@ -1892,7 +1892,7 @@ def test_list_tables(self):
         self.assertEquals(spark.catalog.listTables(), [])
         self.assertEquals(spark.catalog.listTables("some_db"), [])
         spark.createDataFrame([(1, 1)]).createOrReplaceTempView("temp_tab")
-        spark.sql("CREATE TABLE tab1 (name STRING, age INT) USING parquet")
+        spark.sql("CREATE TABLE tbl1 (name STRING, age INT) USING parquet")
```

Comment:
python test failed 
```
the location('file:/home/jenkins/workspace/SparkPullRequestBuilder/spark-warehouse/tab1') of table('`default`.`tab1`') already exists.;
```
the location of tab1 does not deleted , I have tried to found out in which test case it forget to delete it(search all test cases containing `tab1` and run it), but they are all ok to delete the location of tab1, so here we change the table name to work around it.